scanpy/tests/test_neighbors.py

```diff
@@ -114,41 +114,78 @@ def neigh() -> Neighbors:
     return get_neighbors()
 
 
+@pytest.mark.parametrize('method', ['umap', 'gauss'])
+def test_distances_euclidean(mocker, neigh, method):
+    """umap and gauss behave the same for distances.
+
+    They call pynndescent for large data.
+    """
+    from pynndescent import NNDescent
+
+    # When trying to compress a too-small index, pynndescent complains
+    mocker.patch.object(NNDescent, 'compress_index', return_val=None)
+
+    neigh.compute_neighbors(n_neighbors, method=method)
+    np.testing.assert_allclose(neigh.distances.toarray(), distances_euclidean)
+
+
+@pytest.mark.parametrize(
```

Comment:
Could we add a case passing in an actual transformer class? Maybe one of the sklearn.neighbors provided ones?

Reply:
sure!